### ISSUE TITLE
Still developing or abandoned?

### ISSUE THREAD

**Issue by kaangokdemir** [bloodysummers/headrush-desktop#1]
Cool project! Are you still developing since there have been no commits for 9 months?

If yes, I can contribute to some features like

Save / Discard

<img width="1112" alt="Screenshot 2024-01-27 at 17 04 28" src="https://github.com/bloodysummers/headrush-desktop/assets/51219535/51166753-25d3-4917-a7af-7f76394cf252">

or Rename a rig.

Please let me know. Thanks! @bloodysummers @ldiaz-occ

**Comment by SlyDeath** [comment]
Is it dead?

**Comment by bloodysummers** [comment]
I haven't included any new features, I was stuck with the settings for amps, cabs and IRs and left it for a while, but it would be great if you want to contribute.

**Comment by CatsEyeXI** [comment]
Let's get this going guys!! Someone want to fire up a discord server so we can discuss the development of this app?

**Comment by mnajle** [comment]
I Would love to contribute. Actually, I was starting to create my own project for the same application and then found this that looks really well.

**Comment by jerdog** [comment]
Did anything come from the conversation about discord + community development?

**Comment by bloodysummers** [comment]
No, if someone wants to start a discord server or anything I will gladly join, I really want to continue the project.
